packages/presentation/lib/screen/home/widgets/home_shimmer.dart

```diff
@@ -1,15 +1,19 @@
 import 'package:flutter/material.dart';
+import 'package:presentation/config/dimens/dimens.dart';
 import 'package:presentation/config/theme/app_colors.dart';
 import 'package:presentation/screen/home/widgets/shimmer_container.dart';
 import 'package:shimmer/shimmer.dart';
 
 class HomeShimmer extends StatelessWidget {
-  const HomeShimmer({super.key}) ;
+  const HomeShimmer({super.key});
 
   @override
   Widget build(BuildContext context) {
     final size = MediaQuery.of(context).size;
     final width = size.width;
+    int itemCountLength = 6;
+    int crossAxisCountSize = 2;
+    int flexSize = 2;
```

Comment:
make it static const if these variables will not change

Reply:
corrected here and in other files